common/static/common/js/discussion/utils.js

```diff
@@ -386,8 +386,7 @@
                         /*
                          bug fix, ordering is off
```

Comment:
@noraiz-anwar , I'm more concerned why that comment is there. It looks like someone tried to fix a similar bug, so I wanted to know why someone added that comment? If they were trying to fix an ordering bug, why do we still see an ordering bug?